php/libraries/NDB_Client.class.inc

```diff
@@ -98,6 +98,9 @@ class NDB_Client
 		WHERE cs.Name=:nm",
             array("nm" => "CSPAdditionalHeaders")
         );
+        if (count($config_additions) < 1) {
```

Comment:
I don't know if this will work. count() returns the number of elements in an array, but CSPAdditionalHeaders is supposed to be a string (unless it's not defined, in which case pselect returns an empty array..)

Maybe this should be if empty() instead?